src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -429,106 +427,96 @@ limitations under the License.
       <description>ACCompressorType</description>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute side="server" code="0x0044" define="AC_ERROR_CODE" type="ACErrorCodeBitmap" min="0x0" max="0x1F" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0044" define="AC_ERROR_CODE" type="ACErrorCodeBitmap" writable="true" default="0" optional="true">
       <description>ACErrorCode</description>
       <access op="write" privilege="manage"/>
     </attribute>
     <attribute side="server" code="0x0045" define="AC_LOUVER_POSITION" type="ACLouverPositionEnum" min="0x0" max="0x5" writable="true" default="0" optional="true">
       <description>ACLouverPosition</description>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="temperature" writable="false" default="0x8000" optional="true" isNullable="true">ACCoilTemperature</attribute>
-    <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="ACCapacityFormatEnum" min="0x00" max="0xFF" writable="true" default="0" optional="true">
-      <description>ACCapacityformat</description>
+    <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="temperature" optional="true" isNullable="true">ACCoilTemperature</attribute>
+    <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="ACCapacityFormatEnum" min="0x00" max="0x00" writable="true" default="0" optional="true">
+      <description>ACCapacityFormat</description>
```

Comment:
This change right here is an API break, which is probably not OK without backwards compat shims of various sorts in out various APIs.  Please revert the naming change (ACCapacityformat to ACCapacityFormat) and we should figure out in a separate discussion whether to take that pain or just "fix" the spec.

This part is the real "changes requested" bit.

Reply:
In this case, the spec was always capitalized correctly; this typo was introduced in #18594.

In the HVAC call, no one had ever implemented any of these attributes, and we intend to deprecate them at some point.

In the meantime, reverted to ACCapacityformat.

Reply:
Yeah, I'm not saying it's good we have the typo or anything (and I feel really bad that I introduced it!); it's just mechanically it's an API break at this point and we can't just do that.